service/udp.go

```diff
@@ -228,6 +226,27 @@ func (s *udpService) Serve(clientConn net.PacketConn) error {
 	return nil
 }
 
+// Given the decrypted contents of a UDP packet, return
+// the payload and the destination address, or an error if
+// this packet cannot or should not be forwarded.
+func (s *udpService) parsePacket(textData []byte) ([]byte, *net.UDPAddr, *onet.ConnectionError) {
```

Comment:
It would be nice to convey that there's some validation going on here. Perhaps `parseAndValidatePacket()` or `validatePacket()`?

Reply:
OK, changed to `validatePacket()`